lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -19,7 +20,7 @@
 @attr('shard_3')
 @ddt.ddt
 @patch.dict('django.conf.settings.FEATURES', {'ENABLE_SPECIAL_EXAMS': True, 'MILESTONES_APP': True})
-class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin):
+class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin, ModuleStoreTestCase):
```

Comment:
Side note for anyone interested.  [Super Considered Super](https://www.youtube.com/watch?v=EiOglTERPEo) was a great talk I saw Raymond Hettinger give that explains Python's inheritance rules very clearly.  Worth 45 minutes of your time (30 if you skip the Q&A).